backend/app/controllers/spree/admin/refunds_controller.rb

```diff
@@ -8,6 +8,26 @@ class RefundsController < ResourceController
 
       rescue_from Spree::Core::GatewayError, with: :spree_core_gateway_error
 
+      def create
+        @refund.attributes = refund_params
+        if @refund.save
+          @refund.perform!
+          flash[:success] = flash_message_for(@refund, :successfully_created)
+          respond_with(@refund) do |format|
+            format.html { redirect_to location_after_save }
+            format.js   { render layout: false }
```

Comment:
We don't have a js view here, so we can remove this line. (it made sense in ResourceController because some controllers would have a JS view for create)